drivers/ACPI.c

```
@@ -180,18 +187,17 @@ void ACPIShutdown(void) {
         // Wait a bit
         delay(10);
     }
-    
-    // Fallback methods
-    PrintKernel("ACPI: Trying QEMU shutdown\n");
-    outw(0x604, 0x2000);
-    
+
     PrintKernel("ACPI: Trying Bochs shutdown\n");
     outw(0xB004, 0x2000);
```

Clang (14.0.6)

[warning] 0xB004 is a magic number; consider replacing it with a named constant

(cppcoreguidelines-avoid-magic-numbers,readability-magic-numbers)